File: FreeCodeCamp/Javascript/frankenSplice.js
// https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/slice-and-splice
// Reprak11
function frankenSplice(arr1, arr2, n) {    
    return arr2.slice(0,n).concat(arr1).concat(arr2.slice(n));
}

console.log(frankenSplice(["claw", "tentacle"], ["head", "shoulders", "knees", "toes"], 2));

File: FreeCodeCamp/Python/frankenSplice.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/slice-and-splice
# Reprak11

def frakenSplice(arr1, arr2, n):
    return arr2[:n] + arr1 + arr2[n:]

if __name__ == "__main__":
    print(frakenSplice(["claw", "tentacle"], ["head", "shoulders", "knees", "toes"], 2))
